### ISSUE TITLE
Adding a song by double-clicking a suggestion duplicates it

### ISSUE THREAD

**Issue by PaulDance** [matthias4217/jukebox-ultra-nrv#52]
The latest version of the commit, as of writing this comment, seems to add a song twice when selected by a double-click from the suggestions:

![capture](https://user-images.githubusercontent.com/35344098/57737209-14aedc80-76ab-11e9-8236-1bbfe6c522ad.png)

**Comment by PaulDance** [comment]
As I was wrong on the bug at first, as double-clicking a suggestion seemed to be the way to do before, but I'm not sure. Maybe consider adding a sort of lock to avoid accepting double or more clicks on a suggestion ?

**Comment by matthias4217** [comment]
Indeed, we plan to rework the UI a bit, so that the user may see the URL of songs (#37) and the interface would be usable with a keyboard only (#32). I guess I'll put a delay between a user is able to add to songs at the same moment.
